environments/production/aws-resources.yml

```diff
@@ -7,6 +7,7 @@ couch12-production: 10.202.40.52
 couch13-production: 10.202.40.154
 couch14-production: 10.202.40.216
 couchproxy0-production: 10.202.40.162
+djangomanage0-production: 10.202.10.185
```

Comment:
Was this intentional to add?

Reply:
Good question. Looks like that machine must still be running despite having been replaced and not showing up in our inventory. I've checked previously with people that no one's using it, so I'm going to delete it and remove this line.